### PR TITLE
Next Landing Page Rev

### LINKED ISSUE #14: Landing Page---SVG Circles Aliasing
image

- [ ] 1. Line weight of the circles is too small, the tops and bottoms are fading out

- [x] 4. reduce padding by 1/3 or so.

### LINKED ISSUE #15: Landing Page --- Hero Text
image

- [x] header —> see comp on the right (obviously would need to be match the IDL type scale)

- [x] while it appears lower on the page, i think the header wants to be “What is IBM Design Research?”

### LINKED ISSUE #16: Landing Page --- Header size, Moving copy
image

- [x] 3. move “why do it” below the the black header box, move “IDR definition” to ‘why do it’ location
- [x] 5. ~is there some better affordance than “read more?”~
- [x] 6. header size? seems too small
- [x] 7 .change title to ethics

- [x] copy edit: 

Design research is at the heart of Enterprise Design Thinking. If you're not practicing design research, you're not working in the Loop. Whole team design research uncovers insights and informs the experiences we create for people.

Good design is driven by good research and it turns out good design really is good business. Learn more about the business impact of Enterprise Design Thinking and design research in this Forrester report.

### LINKED ISSUE #26: Landing Page --- Mobile Designs
I need mobile designs.

### LINKED ISSUE #27: Landing Page --- Mid-page Navigation Menu - Pictograms
image

- [ ] Keep aspect ratio on resize
- [ ] Use hover tokens
- [ ] Focus state

tiles: move pictogram flush bottom left, move arrow flush bottom right (arrow should be slightly larger, Cris G. to provide)

### LINKED ISSUE #28: Landing Page --- Mid-page Navigation Menu - Aspect Ratio, Mobile Responsiveness
image

research practice tiles; maintain aspect ratio across breakpoints (always stay squares), when tiles stack, the spacing should be even spacing (thin)

image

### LINKED ISSUE #30: Landing Page --- Copy change
image

### LINKED ISSUE #38: Overview Pages --- Nav Tiles
*old*
image

image

**new**
image

image

Follow the rectangle pattern from Carbon

- Update aspect ratio for tiles
- tiles: header: use same size as "Drive innovation through evidence"
- update line height for tiles (get Carbon tokens)
- tiles: pictogram flush bottom left, remove "read more", and move right arrow flush bottom right

### LINKED ISSUE #40: Plan for Success --- Files
image

- [x] Change "Learn more" to "Get started" and update to H3 (or H4)
- [x] capitalize R in IBM Design Research

### LINKED ISSUE #45: "Make to Learn" link text updates
@sstrubberg I saw the slack about the links not being entered into markdown correctly. Here is what they should correspond to:

https://www.ibm.com/design/thinking/page/toolkit/activity/paper-prototypes
Please link "paper prototype" in the paragraph under the 'Make with your hands' section.

https://www.ibm.com/design/thinking/page/toolkit/activity/cognitive-walkthrough
Please link "use what you make" under the 'Consider others' section.

https://www.ibm.com/design/thinking/topic/Practitioner/Reinvention/01/01/01/5
Please link "make with your team" in the paragraph under the 'Make with your hands' section.

### PR DESCRIPTION
**Pull request by sstrubberg**
Staging --> https://deploy-preview-44--idr-public.netlify.com/

Updated the following Carbon Components..

```
    "@carbon/colors": "10.2",
    "@carbon/grid": "10.2.0",
    "@carbon/icons-react": "10.2",
    "@carbon/import-once": "10.2.0",
    "@carbon/layout": "10.2.0",
    "@carbon/type": "10.2.0",

```

Lots of work on the landing page.

- New hero image and text
- New tiles for navigation
- Updated copy
- Cleaned up CSS in homepage.scss

Closes #45 
Closes #40 
Closes #38 
Closes #30 
Closes #28 
Closes #27 
Closes #26 
Closes #16 
Closes #15 
Closes #14

### CODE REVIEW

File: src/components/Homepage/homepage.scss [hunk 1]
```diff
@@ -90,106 +122,114 @@
...
+  outline: $carbon--magenta-50 solid 1px;
 }
 
-.homepage--footer__info {
-  display: flex;
-  flex-direction: column;
-  align-items: flex-start;
+.homepage--tile-border:hover {
+  background-color: $carbon--magenta-50;
+  transition: 150ms cubic-bezier(0.2, 0, 0.38, 0.9);
```
Comment (alisonjoseph) [inline src/components/Homepage/homepage.scss]: Carbon has some pre-defined transitions you can use `transition: $transition--base $bx--standard-easing;` for example. Docs are on the Motion page near the bottom https://www.carbondesignsystem.com/guidelines/motion/basics#duration-tokens

**Review by alisonjoseph — APPROVED** [review]
Look good on my end

not related to this PR -- line height looks weird on these tiles, they probably need a type token specified maybe? 

Screen Shot 2019-06-07 at 2 10 36 PM


Also, will need to update the switcher at some point to get the latest links. https://github.com/carbon-design-system/gatsby-theme-carbon/issues/143

Not sure how much time you have, but might be worth it to update this to use the Gatsby theme.

**Comment by sstrubberg** [comment]
> Look good on my end
> 
> not related to this PR -- line height looks weird on these tiles, they probably need a type token specified maybe?
> 
> Screen Shot 2019-06-07 at 2 10 36 PM
> 
> Also, will need to update the switcher at some point to get the latest links. [carbon-design-system/gatsby-theme-carbon#143](https://github.com/carbon-design-system/gatsby-theme-carbon/issues/143)
> 
> Not sure how much time you have, but might be worth it to update this to use the Gatsby theme.

Those tiles are going to get re-worked when @crisgonz90 gives me new specs to work from.

**Comment by CrisGonzo** [comment]
Can we add a 1px or 2 between the tiles in the overview pages. 
 image

**Comment by eleanorbartosh** [comment]
agreed that line height for the descriptions in the boxes feels a bit squished.

**Comment by eleanorbartosh** [comment]
My only other comment is the interior pages feel very devoid of color, especially after the homepage has so much pink and interaction through the colored hover states. @CrisGonzo what was your thought here?

**Comment by sstrubberg** [comment]
@CrisGonzo @eleanorbartosh

**Review by CrisGonzo — APPROVED** [review]

**Comment by eleanorbartosh** [comment]
into it

**Pull request merged**